nixos/modules/services/networking/rosenpass.nix

```
@@ -0,0 +1,220 @@
+{ config
+, lib
+, options
+, pkgs
+, ...
+}:
+with builtins;
+with lib; let
```

```suggestion
with lib; let
```
with lib is kinda accepted while not being great but builtins are rarely used, so we can write the prefix in the little places it is used.

Also this is a good example why multiple withs over the same scope are bad.